integrations/parsely.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Integration: Parse.ly
+ *
+ * @package Automattic\VIP\Integrations
+ */
+
+namespace Automattic\VIP\Integrations;
+
+/**
+ * Loads Parse.ly VIP Integration.
+ *
+ * @private
+ */
+class ParselyIntegration extends Integration {
+	/**
+	 * Loads the plugin.
+	 *
+	 * @param array $config Configuration for this integration.
+	 *
+	 * @private
+	 */
+	public function load( array $config ): void {
+		// Do not load plugin if already loaded by customer code.
+		if ( class_exists( 'Parsely\Parsely' ) ) {
+			return;
+		}
+
+		// Activating the integration via setting constant whose implementation is already
+		// handled via Automattic\VIP\WP_Parsely_Integration (ideally we should move
+		// all the implementation here such that there will be only one way of managing
+		// the plugin).
+		define( 'VIP_PARSELY_ENABLED', true );
```

Comment:
Won't this cause our activation stats to be skewed because everything activated through this integrations framework will show up as `ENABLED_CONSTANT`?

Reply:
As far as I know this is what we wanted? i.e. we introduced this constant at the start of P100 project.

Reply:
We should still be able to differentiate between the different ways the plugin will be activated like whether it's through platform config or through customer code.

We can leave this for now though and handle it separately when we deprecate the `WP_Parsely_Integration` implementation.